README.md

```diff
@@ -0,0 +1,190 @@
+# ckanext-ontario_theme
+
+Theme for Ontario ckan including:
+
+* metadata schema
+* forms
+* templates and design
+* validation for CSV resources
+
+## Requirements
+Related projects, repositories, branches and CKAN plugins:
+| Project | Github group/repo | Branch       | Plugins | 
+| --- | -------- | ---------- | ----------- | 
+| CKAN   | [ckan/ckan](https://github.com/ckan/ckan/)      | ckan-2.9.x | N/A      | 
+| Scheming extension   | [open-data/ckanext-scheming](https://github.com/open-data/ckanext-scheming)     | master | `scheming datasets`       |  
+| Fluent extension   | [open-data/ckanext-fluent](https://github.com/open-data/ckanext-fluent)     | master | `fluent`       | 
+| Validation extension   | [fork of frictionlessdata/ckanext-validation](https://github.com/ongov/ckanext-validation)     | develop | `validation`       | 
+ 
+***
+## ckanext-validation Integration
+
+This branch contains a redesign of the workflow for adding and editing resources where
```

Comment:
i think we need to write it as if writing for `main` branch and not make it branch specific. The images and explanation will be useful to put into `main` as well

Reply:
done